src/connectors/snowflake/trulens/connectors/snowflake/utils/snowflake_server_side_evalutation_artifacts.py

```diff
@@ -0,0 +1,202 @@
+import os
+import re
+
+from snowflake.snowpark import Session
+
+_ZIPS_TO_UPLOAD = [
+    "snowflake_sqlalchemy.zip",
+    "trulens_connectors_snowflake.zip",
+    "trulens_core.zip",
+    "trulens_feedback.zip",
+    "trulens_providers_cortex.zip",
+]
+
+_STORED_PROCEDURE_PYTHON_CODE = """
+import _snowflake
+import trulens.providers.cortex.provider
+from trulens.core import TruSession
+from trulens.core.schema.feedback import FeedbackRunLocation
+
+
+def run(session):
+    # Set up sqlalchemy engine parameters.
+    conn = session.connection
+    engine_params = {}
+    engine_params["paramstyle"] = "qmark"
+    engine_params["creator"] = lambda: conn
+    database_args = {"engine_params": engine_params}
+    # Ensure any Cortex provider uses the only Snowflake connection allowed in this stored procedure.
+    trulens.providers.cortex.provider._SNOWFLAKE_STORED_PROCEDURE_CONNECTION = (
+        conn
+    )
+    # Run deferred feedback evaluator.
+    db_url = _snowflake.get_generic_secret_string("trulens_db_url")
+    session = TruSession(
+        database_url=db_url,
```

Comment:
If we are passing connection to the engine, why do we still need `db_url`?

Reply:
Yeah talked offline about this, but it turns out I don't need to anymore, so thanks for the suggestino haha.